src/components/organisms/SchedulePageModal/utils.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  differenceInMinutes,
+  endOfDay,
+  getUnixTime,
+  isWithinInterval,
+  max,
+  min,
+  startOfDay,
+} from "date-fns";
+
+import { Room } from "types/rooms";
+import { PersonalizedVenueEvent, VenueEvent } from "types/venues";
+import { MyPersonalizedSchedule } from "types/User";
+
+import { WithVenueId } from "utils/id";
+import { eventEndTime, eventStartTime } from "utils/event";
+import { isTruthy } from "utils/types";
+
+import { RoomWithEvents } from "components/molecules/Schedule/Schedule.types";
+
+export const isEventLaterThisDay = (date: number | Date) => (
+  event: VenueEvent
+) =>
+  isWithinInterval(date, {
+    start: startOfDay(eventStartTime(event)),
+    end: eventEndTime(event),
+  });
```

Comment:
This could probably be moved into `utils/events`?

Reply:
moved: https://github.com/sparkletown/sparkle/pull/1302/commits/d55f4a890d70697cc65f57345c7afcbf06034d5f